CHANGELOG-7.x.md

```diff
@@ -6,6 +6,11 @@ The most notable change is that you need PHP 8.1/8.2 to use the new version. The
 the SDK has introduced some breaking changes concerning parameter types, which should not affect your
 project in most cases.
 
+* Dropped support for not actively supported PHP versions. Supported PHP versions are 8.1.x and 8.2.x
+* Added direct dependency on `firebase/jwt`.
```

Comment:
I wouldn't add dependency changes to the changelog. The firebase-jwt library was already included via the Google Auth Library anyway 😅 
```suggestion
```